top_films.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from pyspark.sql import functions as f
+
+import connections as con
+
+titles_info = con.read_tsv('title.basics.tsv')
+
+rating_info = con.read_tsv('title.ratings.tsv')
+
+title_rating_info = titles_info.join(rating_info,
+                                     titles_info.tconst == rating_info.tconst, 'inner') \
+    .drop(titles_info.tconst) \
+    .where((f.col('titleType') == 'movie')
+           & (f.col('numVotes') >= 100000)) \
+    .orderBy(f.col('averageRating').desc(),
+             f.col('numVotes').desc())
+
+
+def top_films_of_all_times():
+    """
+    Find the best films of all times"
+    """
+    top_films_of_all_times = title_rating_info \
+        .select('tconst', 'primaryTitle', 'numVotes',
+                'averageRating', 'startYear') \
+        .limit(100)
```

Comment:
you do `limit` everywhere, maybe it's better to do it in `write_csv()` with parameter? you can set default parameter =100 but change from task to task if needed